interop-switch/values.yaml

```diff
@@ -26,6 +26,8 @@ config:
   PSQL_DATABASE: 'interop_switch'
   PSQL_USER: 'interop_switch'
   PSQL_PASSWORD: 'cVq8iFqaLuHy8jjKuA'
+  ML_API_ADAPTER_HOST: 'test-ml-api-adapter.mojaloop.live'
+  ML_API_ADAPTER_PORT: 80
```

Comment:
I suggest that this be dynamically configured based on the deployment, and not hardcoded :(